src/lib/rule-finder.js

```diff
@@ -108,6 +108,6 @@ function RuleFinder(specifiedFile, options) {
   this.getUnusedRules = () => getSortedRules(unusedRules);
 }
 
-module.exports = function (specifiedFile, options = {}) {
+module.exports = function (specifiedFile, options) {
```

Comment:
@ljharb This is well-covered in the tests; the problem is that specifying the default in both places means that the one on the `RuleFinder` constructor isn't covered by tests because it's only ever called through this exported function.